File: UVa/java/UVa_11172_RelationalOperators.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int tests, a, b;
        tests = scanner.nextInt();

        while(tests-- > 0) {
            a = scanner.nextInt();
            b = scanner.nextInt();

            if(a < b) System.out.println("<");
            else if (a > b) System.out.println(">");
            else System.out.println("=");
        }
    }
}

File: UVa/kotlin/UVa_11172_RelationalOperators.kt
import kotlin.io.*
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)

    var tests = scanner.nextInt()
    var a: Int
    var b: Int
    while(tests-- > 0) {
        a = scanner.nextInt()
        b = scanner.nextInt()

        if (a > b) println(">")
        else if (b > a) println("<")
        else println("=")
    }
}